packages/sdk/src/evm/core/classes/airdrop-erc721.ts

```diff
@@ -38,7 +33,11 @@ export class Airdrop721<T extends IAirdropERC721 | AirdropERC721>
    *******************************/
 
   drop = /* @__PURE__ */ buildTransactionFunction(
-    async (tokenAddress: string, tokenOwner: string, contents: Airdrop721Content[]): Promise<Transaction<{recipient: string}[]>> => {
+    async (
+      tokenAddress: string,
+      tokenOwner: string,
+      contents: Airdrop721Content[],
+    ): Promise<Transaction<{ failedRecipient: string; tokenId: number }[]>> => {
```

Comment:
do a type for this, and should also have the success ones
{
successfulDrops: { ... }[]
failedDrops: { ... }[]
}